### PR TITLE
Initial fix

### PR DESCRIPTION
**Pull request by dracc**
Fix all indentations and repair some minor bugs.
- Make sure all lines end with a semicolon
- `/settime` didn't break, continued into `/setweather`
- Move all teleportation destinations into the table and remove the switch case
- Shorten some lines in a halfhearted attempt to adhere to the 80 char rule
- Move emotes into a table, much like the teleportation destinations

### CODE REVIEW

File: scripts/module/teleporter.nut [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+teleporters["jd"] <- Vector( 232.2453, -1279.0906, 12.0712 );
+teleporters["cc"] <- Vector( -657.2713, -1478.9364, 13.8016 );
+teleporters["gd"] <- Vector( 335.7215,-233.5262,31.1983 );
+teleporters["jcj"] <- Vector( -658.3732,761.5365,11.5527 );
+teleporters["jy"] <- Vector( -1718.5946, -280.8699, 14.8683 );
+teleporters["dyy"] <- Vector( 7.0703, 964.4935, 10.4632 );
+teleporters["y"] <- Vector( 557.677, -886.057, 432.612 );
 
 function lookup_location(location) {
   local result = false;
```
Comment (JayFoxRox) [inline scripts/module/teleporter.nut]: No longer necessary

File: scripts/module/emote.nut [hunk 2]
```diff
@@ -1,222 +1,227 @@
...
-      MessagePlayer("[#6666FF]READ ME -> The symbol ^ means the anim has more options. Example: /a dance, /a dance 2, /a dance 3.",player);
+      MessagePlayer("[#6666FF]Anims' list: [#FFFF81](/a) stop, watcha, lean, " +
+		    "talk, wave, die^, fall^, fallback^, :o, apply, bu!, arg^" +
+		    ", carwback, wback, elbow^, rollback, boss, crawling," +
+		    "rollinfront, falling, roll, beware, idk, look, dumb," +
+		    "aim^, drown, handsup, wut, fu, phone, sit, stand, sell," +
+		    "hm?, onfoot, db^, heal, lay, bored, riot, dance, drunk",
+		    player);
+      MessagePlayer("[#6666FF]READ ME -> The symbol ^ means the anim has more options." +
+		    "Example: /a dance, /a dance 2, /a dance 3.",player);
```
Comment (JayFoxRox) [inline scripts/module/emote.nut]: Just do the same as for `teleporters`.

auto-generate help message and just keep a long list with group and animation index.

```
emotes <- {
  "stop": [0, 29],
  "...": [...],
}
```

File: scripts/module/hyper.nut [hunk 3]
```diff
@@ -2,54 +2,64 @@
...
+    else if (id == HYPER_DOWN) {
+      player.Pos.z -= 15;
+    }
+    else {
+      local rad = player.Angle, pos = player.Pos;
+      if (id == HYPER_FORWARD);
+      else if (id == HYPER_BACK) {
+	rad -= 3.14;
+      }
+      else if (id == HYPER_LEFT) {
```
Comment (JayFoxRox) [inline scripts/module/hyper.nut]: I prefer `} else` on the same line; all other instances too

File: scripts/scripts.nut [hunk 4]
```diff
@@ -31,255 +31,235 @@ function scripts_handle(name, ...) {
...
-  return final_result
+  return final_result;
 }
 
 class Hooks {
 
-function onServerStart() {
-  scripts_handle("onServerStart")
-}
+  function onServerStart() {
```
Comment (JayFoxRox) [inline scripts/scripts.nut]: `class` acts as a namespace. formatting of these hooks is also only changed once implemented.

I suggest to not edit this portion of the file.

File: scripts/module/hyper.nut (hunk 3 above)
Comment (dracc) [reply scripts/module/hyper.nut]: I told you I'd convert to `stroustrup` style, he does not use trust in the OTB.

File: scripts/module/emote.nut [hunk 5]
```diff
@@ -1,227 +1,44 @@
...
+  "stand" : [0, 168], "sell" : [0, 171], "hm?" : [0, 172],
+  "onfoot" : [3, 192], "db" : [3, 201], "db 2" : [3, 202],
+  "db 3" : [3, 203], "heal" : [24, 214], "lay" : [25, 210],
+  "bored" : [26, 215], "bored 2" : [26, 216], "bored 3" : [26, 217],
+  "bored 4" : [26, 218], "riot" : [27, 219], "riot 2" : [27, 220],
+  "riot 3" : [27, 221], "riot 4" : [27, 222], "riot 5" : [27, 223],
+  "riot 6" : [27, 224], "riot 7" : [27, 225], "dance" : [28, 226],
+  "dance 2" : [28, 227], "dance 3" : [28, 228], "dance 4" : [28, 229],
+  "dance 5" : [28, 230], "dance 6" : [28, 231], "dance 7" : [28, 232],
+  "drunk" : [29, 210] }
```
Comment (JayFoxRox) [inline scripts/module/emote.nut]: nit: One per line would be much better in my opinion

Also this is a `table` not an `array`

File: scripts/module/hyper.nut (hunk 3 above)
Comment (JayFoxRox) [reply scripts/module/hyper.nut]: Who or what is OTB?

I personally think that newline before an `else` is a sin. The newline is quickly interpreted as: "this statement ends here", but then it surprisingly continues. It's *really* bad in my opinion.

It also makes it way too easy to accidentally re-arrange codeblocks to cause issues.
It's similar to the apple `goto fail` mess.

I don't care about style-names or whatever. I care about code being secure / predictable, and learning from past mistakes.

File: scripts/module/taxi_stunting.nut [hunk 6]
```diff
@@ -40,62 +40,67 @@ function onPlayerMove( player, lastX, lastY, lastZ, newX, newY, newZ ) {
...
 
 function createVisual(player, duration) {
-  removeVisual(player.ID)
-  height_cp[player.ID] = CreateCheckpoint(player, player.UniqueWorld, false, height_record_position, ARGB(255, 255, 0, 0), 4);
-  height_ma[player.ID] = CreateMarker(player.UniqueWorld, height_record_position, 5, RGB(255, 255, 0), 26 );  
+  removeVisual(player.ID);
+  height_cp[player.ID] = CreateCheckpoint(player, player.UniqueWorld, false,
+					  height_record_position, ARGB(255, 255, 0, 0), 4);
+  height_ma[player.ID] = CreateMarker(player.UniqueWorld, height_record_position,
+				      5, RGB(255, 255, 0), 26 );
```
[the remark this answers is not in the document]
Comment (dracc) [reply scripts/module/taxi_stunting.nut]: I guess it boils down to personal preference yet again; while using two panes in Emacs on a 1080p display, lines are broken somewhere around 100 characters. This has led to me preferring an 80 char line limit.

**Pull request merged**